WordPress/Classes/Services/PostService.m

```diff
@@ -569,11 +572,27 @@ - (void)updatePost:(AbstractPost *)post withRemotePost:(RemotePost *)remotePost
                 longitudeID = [longitudeDictionary stringForKey:@"id"];
                 publicID = [geoPublicDictionary stringForKey:@"id"];
             }
+            NSDictionary *publicizeMessageDictionary = [self dictionaryWithKey:@"_wpas_mess" inMetadata:remotePost.metadata];
+            publicizeMessage = [publicizeMessageDictionary stringForKey:@"value"];
+            publicizeMessageID = [publicizeMessageDictionary stringForKey:@"id"];
+
+            NSArray *disabledPublicizeConnectionsArray = [self entriesWithKeyLike:@"_wpas_skip_*" inMetadata:remotePost.metadata];
+            for (NSDictionary *disabledConnectionDictionary in disabledPublicizeConnectionsArray) {
+                NSString *dictKey = [disabledConnectionDictionary stringForKey:@"key"];
+                NSNumber *keyringConnectionID = @([[dictKey stringByReplacingOccurrencesOfString:@"_wpas_skip_" withString:@""]integerValue]);
```

Comment:
(Maybe) split this one in two different lines, to ease out reading? (+ perhaps a comment explaining why we need to remove that substring)